### ISSUE TITLE
Bedrock Support

### ISSUE THREAD

**Issue by mikii124** [rossnoah/PerPlayerKit#5]
Name says it

**Comment by rossnoah** [comment]
What would bedrock support entail? Comparability with Geyser or something else?

**Comment by miki132432** [comment]
Yes, sorry I meant geyser. Currently from what I saw it doesn't support right click whilst you're on Bedrock.

**Comment by jd07159** [comment]
Wouldn't it be better to create a /savekit <slot> command instead?

**Comment by rossnoah** [comment]
Definitely an option. I'm not very familiar with bedrock/geyser. 

Which would be a better experience for bedrock players?

**Comment by jd07159** [comment]
You can check the presence of Bedrock clients using the Floodgate API.
And I think using a command wouldn't cause much inconvenience.

**Comment by miki132432** [comment]
Well, /savekit would work too. But having the same experience as java players would be better.

**Comment by jd07159** [comment]
The UUIDs of Bedrock clients connecting via Geyser all start with '00000000', so there's no need to use the Floodgate API. Wouldn't it work to add a conditional statement for the UUID check in addEditLoad and call addEdit accordingly, @rossnoah?

**Comment by rossnoah** [comment]
I have zero experience with bedrock via geyser. I thought @miki132432 was saying there was an issue with the clicks in the GUI not being handled. I don't see how a UUID starting with 00000000 would affect anything.

**Comment by rossnoah** [comment]
Does bedrock not have right clicks or something?

**Comment by mikii124** [comment]
Thought this was completed due to the bedrock support update, but I tested it and it still has not been added
